tooling/noir_js_types/lib/types.ts

```diff
@@ -11,3 +11,8 @@ export interface Backend {
 
   verifyIntermediateProof(proof: Uint8Array): Promise<boolean>;
 }
+
+export type CompiledCircuit = {
+  bytecode: string;
+  abi: object;
+};
```

Comment:
If so this probably doesn't live here, right?

Reply:
Yeah make sense, I'll modify the name and move CompiledCircuit back to noir_js for now

Reply:
Actually I'll open up an issue with these two points and come back to it since it won't be used externally by users for now